docs/articles_en/openvino_workflow/torch_compile.rst

```diff
@@ -68,20 +68,56 @@ To use ``torch.compile``, you need to add an import statement and define one of
          :align: center
 
 
-Environment Variables
-+++++++++++++++++++++++++++
 
-* **OPENVINO_TORCH_BACKEND_DEVICE**: enables selecting a specific hardware device to run the application. 
-  By default, the OpenVINO backend for ``torch.compile`` runs PyTorch applications using the CPU. Setting 
-  this variable to GPU.0, for example, will make the application use the integrated graphics processor instead.
-* **OPENVINO_TORCH_MODEL_CACHING**: enables saving the optimized model files to a hard drive, after the first application run.
-  This makes them available for the following application executions, reducing the first-inference latency.
-  By default, this variable is set to ``False``. Setting it to ``True`` enables caching.
-* **OPENVINO_TORCH_CACHE_DIR**: enables defining a custom directory for the model files (if model caching set to ``True``).
-  By default, the OpenVINO IR is saved in the ``cache`` sub-directory, created in the application's root directory. 
+Options
+++++++++++++++++++++
+
+You can use a dictionary with options for ``torch.compile`` to set the backend device, 
+enable model caching, set the cache directory etc. See the example below:
+
+.. code-block:: python
+
+   model = torch.compile(model, backend="openvino", options = {"device" : "CPU", "model_caching" : True})
+
+You can also set OpenVINO specific configuration options by adding them as a dictionary under ``config`` key in ``options``:
+
+.. code-block:: python
+
+   opts = {"device" : "CPU", "config" : {"PERFORMANCE_HINT" : "LATENCY"}}
+   model = torch.compile(model, backend="openvino", options=opts)
+
+Below is a list of available options:
+
+* ``device`` - enables selecting a specific hardware device to run the application. 
+  By default, the OpenVINO backend for ``torch.compile`` runs PyTorch applications 
+  on CPU. If you set this variable to ``GPU.0``, for example, the application will 
+  use the integrated graphics processor instead.
+* ``model_caching`` - enables saving the optimized model files to a hard drive, 
+  after the first application run. This makes them available for the following 
+  application executions, reducing the first-inference latency. By default, this 
+  variable is set to ``False``. Set it to ``True`` to enable caching.
+* ``cache_dir`` - enables defining a custom directory for the model files (if 
+  ``model_caching`` is set to ``True``). By default, the OpenVINO IR is saved 
+  in the cache sub-directory, created in the application's root directory.
+* Any OpenVINO configuration option. For details, refer to the :ref:`OpenVINO Advanced Features <openvino-advanced-features>`.
+
+.. important::
+   
+   ``device``, ``model_caching``, and ``cache_dir`` have replaced options that were available in the previous release:
```

Comment:
These are going to replace the environment variables in the coming release Karol. These have not been deprecated yet. 
So we can say, setting options through environment variables will be deprecated from the upcoming release. 
Options will replace environment variables

Reply:
```suggestion
   Setting options through environment variables will be deprecated from the upcoming release. The ``device``, ``model_caching``, and ``cache_dir`` options will replace the environment variables:
```

Reply:
I guess the term "in the coming release" makes it a bit ambiguous. The question is: are these new options going to be available in 23.3 for production purposes? 
* If so, we should say that they have replaced the old ones. Even if the old ones are still available (e.g. for backwards compatibility). They are deprecated, and will be removed entirely in future releases. 
* If the environment variables are still the main way to set options for 23.3, then we should change the order of information - first the default, then the way that will be available in future releases...

Reply:
No the options will be the main way to set the parameters but the users can use env variables as well. We can say that options replaced the environment variables and the env variables will be entirely removed from future releases

Reply:
Right, so I suggest putting it like this and including the whole thing in the drop-down:
```suggestion
   The environment variables used in the previous release are still available but are not
   recommended. They will be removed fully in future releases.
   Click to view the deprecated options:
```

Reply:
Yep this sounds good Karol.

Reply:
```suggestion
   The environment variables used in the previous release are still available but are not recommended. They will be removed fully in future releases.
```